alea/model.py

```diff
@@ -239,6 +236,18 @@ def store_data(
             raise ValueError("The number of data sets and data names must be the same")
         toydata_to_file(file_name, _data_list, data_name_list, **kw)
 
+    def set_nominal_values(self, overwrite_static=False, **nominal_values):
```

Comment:
Recover `set_nominal_values`, with an option to overwrite static parameters. And in principle I think we do need a method to overwrite static parameters(bypass the check).